tokens/_options/color.json5

```diff
@@ -1,6 +1,8 @@
 {
   options: {
     color: {
+      docCategory: 'colors',
```

Comment:
I removed `docCategory` and `docExample` from all the global json5 files and put them in the options file. Wasn't sure if this was the right place for it.

Reply:
Correct -  https://github.com/rei/rei-cedar-docs/blob/next/docs/.vuepress/components/TokenSorter.vue

Reply:
So there are a couple alternative ways to do this:

- Change the name of docCategory/docExample and make them more specific across the files, ie "docCategoryChip" and "docExampleChip"
- Create a new file that only has shared variables within the global folder, ie shared.json5

The advantage of the former is that it's clearer where the value is coming from. The advantage of the latter is that you'd only need to edit it in one place.

I think I prefer the former as it is more explicit and easier to understand where the value is coming from even though we'd be sacrificing some DRYness

Reply:
It might be prudent to look at the designs for the new doc sites's token page and design a schema that work well with those?